### DISCUSSION TITLE
Linuxbrew linking fails with incompatible i386 files already on machine

### DISCUSSION THREAD

**Discussion by kod4krome** [Homebrew/discussions#2013] [Linux]
I'm hitting this error with anything I have tried to install.

`ld: i386 architecture of input file '/lib/crti.o' is incompatible with i386:x86-64 output`
`ld: i386 architecture of input file '/lib/crtn.o' is incompatible with i386:x86-64 output`

The machine is running redhat and devops has installed newer versions of programs/components in non-standard locations. I only have user level access

I forced homebrew to use non-standard locations of curl and git, but I do not know how to point to a newer version of gcc or tell it to add options to the linker to handle the older (glibc?) files.

Any help greatly appreciated.  Thanks

**Comment by danielnachun** [comment]
Have you run `brew post-install gcc@5`?  It should copy the correct `crt` files from wherever the host GCC is located into the right directory within `$HOMEBREW/.linuxbrew`.

**Reply by kod4krome** [reply]
Unfortunately I may need a lot of hand-holding here.

```
brew post-install gcc@5
Error: Unknown command: post-install
```

**Reply by danielnachun** [reply]
Shoot, it's `brew postinstall`. Sorry!
